src/config/app.js

```diff
@@ -14,6 +14,7 @@ app.use(cors({ origin: '*' }))
 
 //Rutas
 app.use('/comfystream/api', require('../routes/usuario.routes'))
+app.use('/comfystream/api', require('../routes/evento.routes'))
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7138706)
```suggestion
app.use("/comfystream/api", require('../routes/evento.routes'))
```